src/guides/v2.4/config-guide/redis/redis-pg-cache.md

```diff
@@ -1 +1,188 @@
-../../../v2.3/config-guide/redis/redis-pg-cache.md
\ No newline at end of file
+---
+group: configuration-guide
+title: Use Redis for the Magento page and default cache
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+
+Magento provides command line options to configure Redis page and default caching. Although you can configure caching by editing the `<Magento install dir>app/etc/env.php` file, using the command line is the recommended method, especially for initial configurations. The command line provides validation, ensuring the configuration is syntactically correct.
+
+You must [install Redis]({{ page.baseurl }}/config-guide/redis/config-redis.html#config-redis-install) before continuing.
+
+## Configure Redis default caching {#config-redis-config}
+
+Run the `setup:config:set` command and specify parameters that specific to Redis default caching.
+
+```bash
+bin/magento setup:config:set --cache-backend=redis --cache-backend-redis-<parameter_name>=<parameter_value>...
+```
+
+where
+
+`--cache-backend=redis` enables the Redis default caching. If this feature has already been enabled, omit this parameter.
+
+`--cache-backend-redis-<parameter_name>=<parameter_value>` is a list of key/value pairs that configure the default caching:
+
+|Command line parameter|Parameter|Meaning|Default value|
+|--- |--- |--- |--- |
+|cache-backend-redis-server|server|Fully qualified hostname, IP address, or an absolute path to a UNIX socket. The default value of 127.0.0.1 indicates Redis is installed on the Magento server.|127.0.0.1|
+|cache-backend-redis-port|port|Redis server listen port|6379|
+|cache-backend-redis-db|database|Required if you use Redis for both the default and full-page cache. You must specify the database number of one of the caches; the other cache uses 0 by default.<br/><br/>Important: If you use Redis for more than one type of caching, the database numbers must be different. It is recommended that you assign the default caching database number to 0, the page caching database number to 1, and the session storage database number to 2.|0|
+|cache-backend-redis-password|password|Configuring a Redis password enables one of its built-in security features: the auth command, which requires clients to authenticate to access the database. The password is configured directly in Redis's configuration file, /etc/redis/redis.conf, which you should still have open from the previous step.||
```

Comment:
> which you should still have open from the previous step

This part might be a little bit confusing for a reader. Once I got to this part I was looking for a previous step where we had a deal with `/etc/redis/redis.conf` but didn't found anything specific. So we may clarify this statement. 

Thank you

Reply:
I removed that phrase, as it wasn't needed.